addon/components/bs-contextual-help/element.js

```diff
@@ -96,6 +97,8 @@ export default class ContextualHelpElement extends Component {
 
   offset = [0, 0];
 
+  @trackedRef('popperElement') popperElement;
```

Comment:
Looks like here there is no need for trackedRef, it should be used if node used inside some reactive getter

Reply:
It is used in the `popperModifiers` getter. There is even a inline comment that it should recompute when `popperElement` is set later:

> ```
> // We need popeerElement, so we wait for this getter to recompute once it's available
> ```

I noticed that `addon/components/bs-popover/element.hbs` uses `{{create-ref "popperElement"}}`, while `addon/components/bs-tooltip/element.hbs` uses `{{create-tracked-ref "popperElement"}}`. Is it different by intention?

Reply:
yes, `{{create-tracked-ref "popperElement"}}` mean we have to track Node mutations (using mutation observer and such), `{{create-ref "popperElement"}}` should be used in current case

Reply:
> Is it different by intention?

I don't think so. Seems wrong?

Reply:
Hmm... Initially I wrote everything with non-tracked versions and it did not solve my case. Making that single entry tracked resolved my case. 🤷‍♂️

Reply:
The use of tracked-ref vs. ref seems to be exchanged for overlayElement and popperElement, when you compare popovers and tooltips!

Reply:
Uhm... I've just tested it out, and it turns out that having the `@trackedRef` decorator on `popperElement` is enough for the `popperModifiers` getter (which consumes `this.popperElement`) to start recomputing.

In the template I have replaced `{{create-tracked-ref}}` with `{{create-ref}}` and it still works. Whereas replacing `@trackedRef` with `@ref` breaks it.

Reply:
Seems good now. I didn't really know about `{{create-tracked-ref}}` before, but it seems to cover a different but not needed use case here, when properties of the DOM element itself change, like adding a MutationObserver and the like!? See https://github.com/lifeart/ember-ref-bucket#definition-of-create-tracked-ref-modifiers

But again, we don't need this here, just need the ref-property in the component to be tracked.